db-migrations/01-add-collections/01-add-collections.sql

```diff
@@ -46,7 +46,7 @@ CREATE TABLE IF NOT EXISTS collections (
     versionNumber VARCHAR (255) NOT NULL,
     status TINYINT DEFAULT 0 NOT NULL,
     tag VARCHAR (200),
-    creationDate TIMESTAMP DEFAULT 0,
+    creationDate TIMESTAMP DEFAULT '1970-01-01 00:00:01',
```

Comment:
the unix timestamp of 0 corresponds to `1970-01-01 00:00:00` right? I doubt it matters to be one second off here though.

Reply:
The range described here is what I'm referencing: https://dev.mysql.com/doc/refman/5.7/en/datetime.html

Reply:
ahh, I see. Ok nevermind!